gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
```

Comment:
We prefer to DDOS _ourselves_ these days:
```
distributionUrl=https\://gradleproxy:gradleproxy@software.r3.com/artifactory/gradle-proxy/gradle-5.6.4-all.zip
```

Reply:
Not sure we'd want to do this on something that is open source would we?  I'm assuming the artifactory proxy is private.

Reply:
> Not sure we'd want to do this on something that is open source would we? I'm assuming the artifactory proxy is private.

We already do this for Corda OS, the Corda Gradle plugins and the DJVM. Not every repo in the Artifactory needs a username and password.